app/src/main/java/com/techcourse/AppWebApplicationInitializer.java

```diff
@@ -13,12 +15,27 @@ public class AppWebApplicationInitializer implements WebApplicationInitializer {
     @Override
     public void onStartup(final ServletContext servletContext) {
         final var dispatcherServlet = new DispatcherServlet();
-        dispatcherServlet.addHandlerMapping(new ManualHandlerMapping());
+        initStrategies(dispatcherServlet);
 
```

Comment:
개인적으로 두 줄 정도는 별도로 메서드를 분리하지 않아도 괜찮지 않나 싶어요.
 
```suggestion
        initHandlerMappings(dispatcherServlet);
        initHandlerAdapters(dispatcherServlet);
```

Reply:
나중에 더 추가되면 분리하였습니당 https://github.com/woowacourse/jwp-dashboard-mvc/pull/123/commits/a4e842bce1e0f090eabcd1b81b79cf6857a83d40